core/modules/widgets/fieldmangler.js

```diff
@@ -135,6 +135,10 @@ FieldManglerWidget.prototype.handleAddTagEvent = function(event) {
 			$tw.utils.pushTop(modification.tags,tag);
 			this.wiki.addTiddler(new $tw.Tiddler(tiddler,modification));			
 		}
+	} else if(typeof event.param === "string" && event.param.trim() !== "" && this.mangleTitle.trim() !== "") {
```

Comment:
I think the core function you call should take care of all that, otherwise this code is implemented in a million places.